src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/components/status_message/status_message.tsx

```diff
@@ -132,7 +132,7 @@ export const StatusMessage: React.FC<StatusMessageProps> = ({
         />
       </span>
     );
-  } else if (allIndicesLength) {
```

Comment:
Previously we only showed this message if we knew somebody had indices and still didn't create a pattern that has a match. But now if someone has no indices, they will still see "You can match one of your indices below" even if there is nothing actually listed below -- just want to double-check that this change was intentional.

Reply:
Yes, its intentional. This is solving for the case where there's no indices AND the user has clicked 'create anyway' in the previous view. If they've clicked 'create anyway' they're likely aiming to create an index pattern off a system index...when they don't have any real indices. Seems like a pretty odd corner case.